docker-compose/baker+wallet-proxy+identity-provider.yaml

```diff
@@ -0,0 +1,168 @@
+# Necessary enviornment variables
+# - HOST_ROOT ... an **absolute** path to a directory where ...
+
+version: '3'
+services:
+  # the collector backend that serves data about the nodes
+  # It exposes endpoints at localhost:12000
+  # The two endpoints of interest are
+  # - GET /nodesSummary
+  # - GET /nodesBlocksInfo
+  collector_backend:
+    image: concordium/dev-node:latest
+    environment:
+    - RUST_BACKTRACE=1
+    - RUST_LOG=info
+    entrypoint: ["/node-collector-backend",
+      "--listen-port", "10000",
+      "--listen-address", "0.0.0.0"
+    ]
+    ports:
+    - 12000:10000
+  # the node to which the wallet-proxy connects.
+  # It is also the only baker in this configuration.
+  baker_reporter:
+    image: concordium/dev-node:latest
+    volumes:
+      # volume mount host path must be relative to **this** docker-compose file,
+      # or an absolute path.
+      - ${HOST_ROOT}/baker-reporter:/var/lib/concordium/
+    depends_on:
+    - postgresql
+    entrypoint: ["/concordium-node",
+       "--no-dnssec",
+       "--no-bootstrap",
+       "--debug",
+       "--config-dir", "/var/lib/concordium",
+       "--data-dir", "/var/lib/concordium",
+       "--baker-credentials-file", "/var/lib/concordium/baker-0-credentials.json",
+       "--bootstrap-node", "bootstrapper:8888",
+       "--transaction-outcome-logging-database-host", "postgresql",
+       "--transaction-outcome-logging-database-port", "5432",
+       "--transaction-outcome-logging-database-name", "concordium",
+       "--transaction-outcome-logging-database-username", "concordium",
+       "--transaction-outcome-logging-database-password", "concordium",
+       "--rpc-server-addr", "0.0.0.0",
+       "--desired-nodes", "0"
+       ]
+    environment:
+    - RUST_BACKTRACE=1
+    - RUST_LOG=info
+    ports:
+    - 10000:10000
+
+  # The collector that posts data to the collector backend to display it.
+  collector:
+    image: concordium/dev-node:latest
+    depends_on:
+    - baker_reporter
+    environment:
+    - RUST_BACKTRACE=1
+    - RUST_LOG=info
+    entrypoint: ["/node-collector",
+      "--collector-url", "http://collector_backend:10000/nodes/post",
+      "--grpc-host", "http://baker_reporter:10000",
+      "--node-name", "reporter-node"
+    ]
+
+  # database where the transaction log is stored for the wallet-proxy.
+  # The database is persisted to `db-volume`
+  postgresql:
+    image: postgres:latest
+    # persist the database through restarts
+    volumes:
+      - db-volume:/var/lib/postgresql/data
+    environment:
+      POSTGRES_PASSWORD: concordium
+      POSTGRES_USER: concordium
+      POSTGRES_DB: concordium
+    ports:
+    - 5432:5432
+
+  wallet_proxy:
+    image: abizjak/wallet-proxy:latest
```

Comment:
This is currently using my own space until images are posted publicly under concordium.